src/js/tabs/pid_tuning.js

```diff
@@ -482,30 +484,35 @@ TABS.pid_tuning.initialize = function (callback) {
         }
 
         if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
-            // Feedforward
+            $('.resetwarning').attr("title", i18n.getMessage("pidTuningResetWarning"));
+        } else {
+            // Previous html attributes for legacy sliders
+            $('.pid_tuning .ROLL input[name="p"]').attr("max", "200");
+            $('.pid_tuning .ROLL input[name="i"]').attr("max", "200");
+            $('.pid_tuning .ROLL input[name="d"]').attr("max", "200");
+            $('.pid_tuning .ROLL input[name="dMinPitch"]').attr("max", "100");
+            $('.pid_tuning .PITCH input[name="p"]').attr("max", "200");
+            $('.pid_tuning .PITCH input[name="i"]').attr("max", "200");
+            $('.pid_tuning .PITCH input[name="d"]').attr("max", "200");
+            $('.pid_tuning .PITCH input[name="dMinPitch"]').attr("max", "100");
+            $('.pid_tuning .YAW input[name="p"]').attr("max", "200");
+            $('.pid_tuning .YAW input[name="i"]').attr("max", "200");
+            $('.pid_tuning .YAW input[name="d"]').attr("max", "200");
+            $('.pid_tuning .YAW input[name="dMinPitch"]').attr("max", "1000");
+            $('#sliderDMaxGain').attr({ "min": "0.5", "max": "1.5" });
+            $('#sliderDTermFilterMultiplier').attr({ "min": "0.5", "max": "1.5", "step": "0.025" });
+        }
+
+        // Feedforward
+        if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
             const feedforwardGroupCheck = $('input[id="feedforwardGroup"]');
-            const PID_FEEDFORWARD = (FC.ADVANCED_TUNING.feedforwardRoll ||
-                FC.ADVANCED_TUNING.feedforwardPitch ||
-                FC.ADVANCED_TUNING.feedforwardYaw);
-
-            feedforwardGroupCheck.prop('checked', PID_FEEDFORWARD !== 0);
+            const PID_FEEDFORWARD = FC.ADVANCED_TUNING.feedforwardRoll || FC.ADVANCED_TUNING.feedforwardPitch || FC.ADVANCED_TUNING.feedforwardYaw;
+            feedforwardGroupCheck.prop('checked', PID_FEEDFORWARD);
```

Comment:
All this check for non existent feedforward switch can be removed, also can be removed switch from html.

Reply:
Added back non expert mode slider limits and also fixed the negative sliders values on 4.3